honeybee_energy/load/setpoint.py

```diff
@@ -51,7 +51,7 @@ class Setpoint(_LoadBase):
         * dehumidifying_setback
     """
     __slots__ = ('_heating_schedule', '_cooling_schedule', '_humidifying_schedule',
-                 '_dehumidifying_schedule')
+                 '_dehumidifying_schedule', '_user_data')
```

Comment:
Remove from `__slots__` since it's already in the parent class.

Reply:
commit: loads cleaned up